### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:

- Added `check_complete.js` with the `checkComplete` function to mark tasks as completed using checkboxes.
- Added `localStorage` functionalities.
- Changed `genHTML` function to generate lined-through text when a task is completed.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   entry: {
     index: './src/index.js',
     print: './src/gen_html.js',
+    check: './src/check_complete.js',
```
Comment (ZawadiSumba66) [inline webpack.config.js:9]: - [ ] I am unable to run your app on my local server. When I do, I get this error and of course, the server does not open: 👇 
imageAre you also facing the same issue, if not could you ask your partner to confirm that?

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi @rpire 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
### Hello @ZawadiSumba66 👋

I'm not having this problem when running the webserver. As you can see it shows permission errors, but it's not keeping the browser from showing the page.



image



The only difference I can see between the picture adjunted to the comment and my code is the `port` specification in `webpack.config.js` I'm not sure if that is what's causing the problem.



image


One of my partners ran it earlier and didn't experienced this problems but I'll ask another one for another review and see if we can address the issue.

Thank you for reviewing my code. I will be waiting for further instructions on how to proceed with this review.

**Comment by ZawadiSumba66** [comment]
@rpire 
I just removed the ` port: 3000,` line that you had added to your code to see if the server would open without it. Unfortunately it did not.  That's why you noticed that change.
By the way, there is a review I missed please remove the node_modules from the root directory and add them to gitignore since it contains many unnecessary files for production.

**Review by dasileker — APPROVED** [review]
### **Hi @rpire** 

### **Project Approved 🎉✅**

### **Optional**

- Please next time be careful about the Webpack, I also couldn't run the project with **npm start**, I did use the watch method.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
### Hello @dasileker 👋

Thank you for reviewing my work. It is running on my side, that is why it is difficult for me to address the issue but, do you think it is a bad set-up of Webpack? And if you think so, do you think it is a good idea to start fresh in a new repository for the next milestone and set up Webpack again?

Once again thank you for your review. I will do it more carefully next time. I only hope to find out why WebServer is not working for other people but it is for me.

Cheers and Happy Coding!

**Comment by dasileker** [comment]
> ### Hello @dasileker 👋
> 
> Thank you for reviewing my work. It is running on my side, that is why it is difficult for me to address the issue but, do you think it is a bad set-up of Webpack? And if you think so, do you think it is a good idea to start fresh in a new repository for the next milestone and set up Webpack again?
> 
> Once again thank you for your review. I will do it more carefully next time. I only hope to find out why WebServer is not working for other people but it is for me.
> 
> Cheers and Happy Coding!

No it's not a bad setup, I think it's a npm issue you could fix by redoing it in a separate folder then add it here.

**Pull request merged**